P5/Exemplars/tei_rdfa.odd

```diff
@@ -0,0 +1,227 @@
+        <moduleRef key="tei"/> <!-- required -->
+        <moduleRef key="header"/> <!-- required -->
+        <moduleRef key="core"/> <!-- required -->
+        <moduleRef key="textstructure"/> <!-- required -->
+        <moduleRef key="gaiji"/>
+        <moduleRef key="verse"/>
+        <moduleRef key="drama"/>
+        <moduleRef key="spoken"/>
+        <moduleRef key="cmc"/>
+        <moduleRef key="dictionaries"/>
+        <moduleRef key="msdescription"/>
+        <moduleRef key="transcr"/>
+        <moduleRef key="textcrit"/>
+        <moduleRef key="namesdates"/>
+        <moduleRef key="figures"/>
+        <moduleRef key="corpus"/>
+        <moduleRef key="linking"/>
+        <moduleRef key="analysis"/>
+        <moduleRef key="iso-fs"/>
+        <moduleRef key="nets"/>
+        <moduleRef key="certainty"/>        
+        <moduleRef key="tagdocs" except="elementSpec moduleRef schemaSpec" />
+        <!-- 
+          From the tagdocs module we exclude the three elements that have
+         a `@prefix` attribute which is both defined locally on
+          elementSpec, moduleRef, and schemaSpec, as well as by our new 
+          class `att.global.analytic.rdfa`. We argue that these tagdocs elements
+          are not going to be used in the context where RDFa support is relevant
+        -->
+
+        <classSpec type="atts" ident="att.global.analytic.rdfa" mode="add">
+          <desc xml:lang="en" versionDate="2020-07-01">provides additional global attributes for
+            embedding RDF following the RDFa Core 1.1 specification</desc>
+          <attList>
+            <attDef mode="add" ident="about">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-about</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a SafeCURIEorCURIEorIRI, used for stating
+                what the data is about (a 'subject' in RDF terminology)</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="content">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-content</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a CDATA string, for supplying
+                machine-readable content for a literal (a 'literal object', in RDF
+                terminology)</desc>
+              <datatype>
+                <dataRef key="teidata.text"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="datatype">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-datatype</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a TERMorCURIEorAbsIRI representing a
+                datatype, to express the datatype of a literal</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="href">
```

Comment:
`@href` and `@src` below are the HTML attributes on `<a>` and `<img>`, for example. We shouldn't add them. The RDFa spec is explaining how these built in HTML attributes interact with RDF—namely they can act in the same way as `@resource`. The corresponding TEI attributes are `@target` and `@url`, though there are many, many more attributes in TEI that could potentially interact with RDFa. See https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-teidata.pointer.html for a rather dizzying list.